.travis.yml

```diff
@@ -1,14 +1,58 @@
 language: r
 cache: packages
-r:
-- 3.3
-- oldrel
-- release
-- devel
 after_success:
 - Rscript -e 'covr::codecov()'
 - bash pkgdown.sh
-env:
-  global:
-    - _R_CHECK_LENGTH_1_LOGIC2_=true
-    - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+
+# matrix to include a windows build this can be removed when 
+# https://github.com/travis-ci/travis-build/pull/1538 is merged in
+# the windows build steps are similar to what's in the PR and the process other
+# oses use
+matrix:
+  include:
+  - os: linux
+    r: 3.3
+    env:
+      global: 
+        - _R_CHECK_LENGTH_1_LOGIC2_=true
+        - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+  - os: linux
+    r: oldrel
+    env:
+      global: 
+        - _R_CHECK_LENGTH_1_LOGIC2_=true
+        - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+  - os: linux
+    r: release
+    env:
+      global: 
+        - _R_CHECK_LENGTH_1_LOGIC2_=true
+        - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+  - os: linux
+    r: devel
+    env:
+      global: 
+        - _R_CHECK_LENGTH_1_LOGIC2_=true
+        - secure: BWvMPmOiTzFNfFz3Ok+WHk6FjNNjK8qV5QohBhS0bgsMgeCrx4Zu3IXp42cXaEzzCe/VEnnEyYYTkZROS3mdLRAlOHnEHvCJMtETOODOK6EFGUFDv+C7AXZyqbdoNPrGrW5QKppHAVbTuL/A9z+tnBaeyG68GhOmOFBk4mciHNc=
+  - os: windows
+    language: shell
+    env:
+      global: _R_CHECK_LENGTH_1_LOGIC2_=true
+    before_install:
+      - choco install -y r.project
+      - export R_VERSION=`ls 'C:\Program Files\R\'`
+      - export PATH=$PATH:';C:\Program Files\R\'$R_VERSION'\bin\x64'
+      - echo 'options(repos = c("https://cloud.r-project.org"))' > ~/.Rprofile.site
```

Comment:
Not that I'm keen on debugging this code, but if we're going to have it, let's make it unoffensive to us. Like, no `c()` around a single string, and put a space after the `;` in the `Rscript.exe -e` lines.

This also looks a lot like what our Makefile does; idk if their windows hosts have make, but if so, maybe we could call that instead.